client/app/bundles/course/group/forms/GroupFormDialog.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useCallback, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { isPristine, submit } from 'redux-form';
+import { Dialog, FlatButton } from 'material-ui';
+import { FormattedMessage } from 'react-intl';
+
+import ConfirmationDialog from 'lib/components/ConfirmationDialog';
+import formTranslations from 'lib/translations/form';
+import modalFormStyles from 'lib/styles/ModalForm.scss';
+
+import { connect } from 'react-redux';
+import actionTypes, { formNames } from '../constants';
+
+const styles = {
+  dialog: {
+    width: '80%',
+    maxWidth: 700,
+  },
+};
+
+const NameDescriptionDialog = ({
```

Comment:
There is an existing FormDialogue component. Could we reuse that?

Reply:
Reused, thanks for pointing it out!